common/config/azure-pipelines/jobs/version-bump-individual.yaml

```diff
@@ -0,0 +1,50 @@
+# The following build handles everything needed to bump the versions of iTwin.js AppUi packages that are using individual version definition.
+
+parameters:
+  - name: VersionPolicy
+    displayName: Version Policy
+    type: string
+    default: "monorepo-individual"
+    values:
+      - monorepo-individual
+
+variables:
+  - group: Caleb - iTwin GitHub Token
+
+trigger: none
+pr: none
+
+pool:
+  vmImage: ubuntu-latest
+
+jobs:
+  - job: Bump
+    displayName: Bump Version
+    steps:
+      - checkout: self
+
+      - task: NodeTool@0
+        displayName: Use Node 16.x
+        inputs:
+          versionSpec: 16.x
+          checkLatest: true
+
+      - bash: |
+          git config --local user.email imodeljs-admin@users.noreply.github.com
+          git config --local user.name imodeljs-admin
+        displayName: Setup Git
+
+      - bash: "node common/scripts/install-run-rush version --bump --version-policy ${{ parameters.VersionPolicy }}"
```

Comment:
Does this correctly bump the version or prompt for any additional input? I'm not sure how rush version --bump works with an "individualVersion" policy.

Reply:
Yes it bumps the versions based on previously gathered `rush change` inputs, i.e. each `rush change` asks to input both the change message and version increment (path/minor/major/).